packages/atoms/src/SelectItem.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { FlagIcon, FlagIconName } from '@trezor/icons';
+
+import { Box } from './Box';
+import { Radio } from './Radio';
+import { Text } from './Text';
+
+export type SelectItemProps = {
+    label: string;
+    value: string | number;
+    onSelect: () => void;
+    isSelected: boolean;
+    isLastChild?: boolean;
+    iconName: FlagIconName;
+};
+
+type SelectItemStyleProps = {
+    isLastChild: boolean;
+    isSelected: boolean;
+};
+const selectItemStyle = prepareNativeStyle(() => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+}));
+
+const underlineSectionStyle = prepareNativeStyle<SelectItemStyleProps>(
+    (utils, { isLastChild, isSelected }) => ({
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        flex: 1,
+        color: utils.colors.gray700,
+        paddingVertical: 20,
+        marginLeft: 10,
+        borderBottomWidth: utils.borders.widths.small,
+        borderColor: utils.colors.gray300,
+        extend: [
+            {
+                condition: isLastChild,
+                style: {
+                    borderBottomWidth: 0,
+                },
+            },
+            {
+                condition: isSelected,
+                style: {
+                    color: utils.colors.black,
+                },
+            },
+        ],
+    }),
+);
+
+export const SelectItem = ({
+    label,
+    value,
+    onSelect,
+    isSelected,
+    iconName,
+    isLastChild = false,
+}: SelectItemProps) => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <TouchableOpacity style={applyStyle(selectItemStyle)} onPress={onSelect}>
+            <FlagIcon name={iconName} />
+            <Box style={applyStyle(underlineSectionStyle, { isLastChild, isSelected })}>
+                <Text numberOfLines={1}>{label}</Text>
+                <Radio value={String(value)} onPress={onSelect} isChecked={isSelected} />
```

Comment:
why String? it's nicer to use just `value.toString()`, but actually we should add `number` as possible value for Radio value prop because sometimes we can pass for example ID which could be number

Reply:
Because right now, radio button only accepts string, so I am casting it to String, but it doesn't really matter too much as the handleSelect happens in Select component. The radio button is kind of dummy here just to be correct semantic style. Select happens on press with the whole row.

Reply:
better add number type to radio `value` prop, it makes sense that number can be used here